src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -29,6 +33,9 @@ export class DotEditContentHtmlService {
     pageModel$: Subject<DotPageContainer[]> = new Subject();
 
     private currentAction: DotContentletAction;
+    private debounceStateSelector = _.debounce((pageLayout: DotLayout) => this.setContaintersSameHeight(pageLayout), 500, {
+        leading: true
+    });
```

Comment:
why create this attribute, I think is better use it directly into the setContaintersChangeHeightListener